.travis.yaml

```diff
@@ -0,0 +1,21 @@
+language: java
```

Comment:
We should run the CI before merging. @dagguh do you know why it didn't start automatically here?

Reply:
In the requests list, we can see that the [PR tried to trigger a build, but it failed](https://travis-ci.com/atlassian/report/requests). I can't link directly to the request, but the error message is:
> Build config file is required via repository settings, but config is empty.

Reply:
Perhaps the file should be called `.travis.yml` rather than `.travis.yaml`

Reply:
It is green now.